epub33/rs/index.html

```diff
@@ -1930,6 +1683,89 @@ <h2>Security and Privacy</h2>
 				<p>The Working Group will have to address the particularity of cross-origin by addressing the question
 					of what the "origin" is for an EPUB Publication.</p>
 			</div>
+
+			<section id="sec-scripted-content-security">
+				<h4>Scripting Considerations</h4>
```

Comment:
I wonder whether the editor's note preceding this section on the origin is still relevant. I believe that has been solved.

Reply:
Actually, I wonder whether the editor's note referring to the pub manifest should still be kept, too. Maybe just to say that this section needs further work overall.